core/scss/components/list-item/mixins.scss

```diff
@@ -11,6 +11,7 @@
   margin-left: $margin-left;
   order: $order;
   flex: $flex-grow $flex-shrink $flex-basis;
+  border: 1px solid transparent;
```

Comment:
Not sure we would want this setting in the list-item mixin:

1. No variables to change it
2. Already have a list-item-color which might more appropriately handle border color.

Reply:
I guess list-item--color not setting border 1px which might not show border in window high contrast mode.